somefile.py

```diff
@@ -42,3 +42,19 @@ def run_command(
 def do_thing():
     print 'thinger'
     print 'thing 2'
+
+import os, re
+
+def thing(self):
+    thing_two('arg1',
+     'arg2')
+    print 'derp'
+
+
+
+
+def thing_two(arg1, arg2):
+    result=arg1*arg2
+    if result <> arg1:
+        print 'derp'
+    ouch =thing / other_thing
```

Comment:
F841 local variable 'ouch' is assigned to but never used
E225 missing whitespace around operator
F821 undefined name 'other_thing'